integration_tests/pages/findASessionPage.ts

```diff
@@ -0,0 +1,22 @@
+import Page from './page'
+import paths from '../../server/paths'
+
+export default class FindASessionPage extends Page {
+  constructor() {
+    super('Track progress on Community Payback')
+  }
+
+  static visit(): FindASessionPage {
+    cy.visit(paths.sessions.show({}))
+
+    return new FindASessionPage()
+  }
+
+  shouldShowSearchForm() {
+    cy.get('h2').contains('Find a session')
+    cy.get('label').contains('Region')
```

Comment:
Thinking about Playwright... I really like the Playwright API for locators like this, for example `page.GetByLabel(`Region`)` would look for an input with the given correctly associated label.